website/docs/reference/resource-properties/arguments.md

```diff
@@ -0,0 +1,145 @@
+---
+title: arguments
+sidebar_label: "arguments"
+id: arguments
+---
+
+import MacroArgsNote from '/snippets/_validate-macro-args.md';
+
+<File name='macros/<filename>.yml'>
+
+```yml
+
+version: 2
+
+macros:
+  - name: <macro name>
+    arguments:
+      - name: <arg name>
+        type: <string>
+        description: <markdown_string>
+
+```
+
+</File>
+
+## Definition
+
+The `arguments` property is used to define the parameters that a macro can accept. Each argument can have a `name`, `type`, and `description`. This helps in documenting the macro and understanding what inputs it requires.
+
+You can use a [behavior flag](/reference/global-configs/behavior-changes#behavior-change-flags) in v1.10. You can define `arguments` in a [`properties.yml`](/reference/resource-properties/arguments#macro-properties) file but they are not enforced by dbt. You can name this file anything and it can placed in the [`macro-paths`](/reference/project-configs/macro-paths) directory (defaults to `["macros"]`).
```

Comment:
```suggestion
You can define `arguments` in a [`properties.yml`](/reference/resource-properties/arguments#macro-properties) file. You can name this file anything and it can placed in the [`macro-paths`](/reference/project-configs/macro-paths) directory (defaults to `["macros"]`). 

You can validate your macro `arguments` using the [`validate_macro_args`](/reference/global-configs/behavior-changes#macro-argument-validation) flag. 
```

Reply:
> You can name this file anything and it can placed in the [`macro-paths`](/reference/project-configs/macro-paths) directory (defaults to `["macros"]`).

this feels relevant to general "macro properties" not to `arguments` specifically

Reply:
<img width="927" alt="Screenshot 2025-04-02 at 5 36 11 PM" src="https://github.com/user-attachments/assets/0f11d5af-ac60-4ea2-9350-aecd1b4e1179" />
repetitive of: